opentech/static_src/src/app/src/components/DetailView/style.scss

```diff
@@ -0,0 +1,19 @@
+.detail-view {
+    margin: 0 -20px;
+
+    @include media-query(tablet-landscape) {
+        display: grid;
+        grid-template-columns: 250px 1fr;
```

Comment:
Do you have any IE11 fallback for this and other uses of `grid-template-columns`